Switch off classic analytics

alicebartlett opened the pull request:
Pivotal: https://www.pivotaltracker.com/story/show/93551518

We're moving over to universal analytics a few apps at a time. This is
the first app to go over. We're keeping the classic tracking code but
sending it to universal analytics instead.
This work should only be merged after this
https://github.com/alphagov/govuk_frontend_toolkit/pull/190 has been
merged, and the front end toolkit has been updated.

I'd like @fofr to check this work as he is the person that spec'ed
the ticket.

fofr commented:
:+1:

fofr commented:
Do performance analyst folks need to prep or check anything pre-merge/deploy? I think someone will need to have setup the Universal custom variables in the analytics profile – I think Ash was dealing with that.

alicebartlett commented:
Thanks @fofr! I've updated the pivotal ticket with your queries and moved the ticket into "owner review" until it is ready for a merge and deploy, when I'll pick it back up.

zilnhoj commented:
I'm looking at this for the Performance Platform and we are preparing our dashboards and collectors to be able to collect the new Organisations custom dimension.
